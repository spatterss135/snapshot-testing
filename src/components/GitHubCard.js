import React from "react"
import {
    Card
} from 'react-bootstrap'
import Button from 'react-bootstrap/Button'

export default function GitHubCard(){
    return(
        <Card style={{ width: '100rem' }}>
            <Card.Img variant='top' src='pic.jpg' alt="Profile photo" style={{'width': '20%'}}></Card.Img>
            <Card.Body>
                <Card.Title>Samuel R. Patterson</Card.Title>
                <Card.Text>I am a student studying fullstack and testing in React</Card.Text>
                <Button variant="primary" href="https://github.com/spatterss135">My Github!</Button>
            </Card.Body>
        </Card>
    )
}