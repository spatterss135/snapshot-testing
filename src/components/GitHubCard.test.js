import renderer from 'react-test-renderer'
import GitHubCard from './GitHubCard'

test('renders a snapshot', () => {
  const tree = renderer.create(<GitHubCard></GitHubCard>).toJSON()
  expect(tree).toMatchSnapshot()
})