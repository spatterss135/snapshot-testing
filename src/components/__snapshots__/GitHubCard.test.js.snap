// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "100rem",
    }
  }
>
  <img
    alt="Profile photo"
    className="card-img-top"
    src="me&millie.jpg"
    style={
      Object {
        "width": "20%",
      }
    }
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Jack H. Ersbo
    </div>
    <p
      className="card-text"
    >
      I am a student fullstack developer who is currently learning about React testing!
    </p>
    <a
      className="btn btn-primary"
      href="https://github.com/jhersbo"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex={0}
    >
      My Github!
    </a>
  </div>
</div>
`;
